lib/addons/nginx/index.ts

```diff
@@ -76,21 +77,11 @@ export class NginxAddOn implements ClusterAddOn {
         this.options = { ...defaultProps, ...props };
     }
 
-    deploy(clusterInfo: ClusterInfo): void {
+    @dependable(['AwsLoadBalancerControllerAddOn'])
```

Comment:
we should probably create an addon enum so we don't have to rely on string here.

Reply:
Did you mean just locally or a global `enum` for all addons?

Reply:
We cannot envision all the add-ons that will be there, including those contributed by the partners or custom add-ons that customers will implement. We could provide a convenience enum (similar to GlobalResources) that makes it easy for the add-ons that we know about. In addition, we now have add-ons that can be applied more than once (such as nested stack) and dependency between those will have to be based on ids provided at design time.